backend/post-service/src/main/java/com/speakapp/postservice/repositories/PostRepository.java

```diff
@@ -13,4 +13,5 @@
 public interface PostRepository extends JpaRepository<Post, UUID> {
 
     Page<Post> findAllByUserIdOrderByCreatedAtDesc(UUID userId, Pageable pageable);
+    Page<Post> findAllByOrderByCreatedAtDesc(Pageable pageable);
```

Comment:
Na pewno działa to? Nie wiem jak łatwo sprawdzić jakie query generuje ta nazwa, podejrzewam, że przez brak podania parametru wszystko generuje się git, ale imo powinno być:
findAllOrderByCreatedAtDesc

Reply:
Ta wersja na pewno działa, sprawdzę tą twoją

Reply:
Niestety ta ładniejsza wersja () generuje błąd przy uruchamianiu post-service, nie potrafi utworzyć takiego query:

Caused by: java.lang.IllegalArgumentException: Failed to create query for method public abstract org.springframework.data.domain.Page com.speakapp.postservice.repositories.PostRepository.findAllOrderByCreatedAtDesc(org.springframework.data.domain.Pageable); No property 'desc' found for type 'Instant'; Traversed path: Post.createdAt

Zostawiam poprzednią wersję

Reply:
Dziwna sprawa...